autoload/neomake/makers/ft/mail.vim

```diff
@@ -0,0 +1,9 @@
+function! neomake#makers#ft#mail#EnabledMakers()
+    return ['proselint']
+endfunction
+
+function! neomake#makers#ft#mail#proselint()
+    return {
+                \ 'errorformat': '%f:%l:%c: %m'
```

Comment:
Should specify a type explicitly, e.g. by prefixing it with `%E` for errors.

Also please add " abort" to the function definitions (check ":Neomake vint" ;)).